src/products/models/__init__.py

```diff
@@ -5,8 +5,8 @@
 from products.models.group import Group
 from products.models.record import Record
 
-Product = Union[Course, Bundle, Record]
-ProductType = Union[Type[Course], Type[Bundle], Type[Record]]
+Product = Union[Course]
+ProductType = Union[Type[Course]]
```

Comment:
Эти две строки получились довольно странными

Reply:
Я сам сомневался, оставить так или убрать и просто курс использовать, подумал, что можно для расширения оставить. Если тебе это тоже кажется сомнительным, то просто на курс типы поменяю тогда

Reply:
```suggestion
ProductType = Type[Course]
```
Я бы пока оставил типы Product, но избавился бы от Union